src/handlers/getCurrentUser.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from 'express-async-router'
+import * as db from '../gateways/storage'
+
+export async function handler(req: express.Request, res: express.Response) {
+  const user = await db.getUser(req.user?.d_id)
+  if (user) {
+    res.status(200).send(user)
+  }
```

Comment:
This handler should exit after calling `send` on the response (line 7), but to make things more clear it's good to add a `return` after calling the `send` if that is not the last line in the whole handler. It makes it more clear that that point is an exit point from the handler.